etc/picongpu/cori-nersc/a100_picongpu.profile.example

```diff
@@ -0,0 +1,78 @@
+# Name and Path of this Script ############################### (DO NOT change!)
+export PIC_PROFILE=$(cd $(dirname $BASH_SOURCE) && pwd)"/"$(basename $BASH_SOURCE)
+
+# User Information ################################# (edit the following lines)
+#   - automatically add your name and contact to output file meta data
+#   - send me a mail on batch system jobs: NONE, BEGIN, END, FAIL, REQUEUE, ALL,
+#     TIME_LIMIT, TIME_LIMIT_90, TIME_LIMIT_80 and/or TIME_LIMIT_50
+export MY_MAILNOTIFY="NONE"
+export MY_MAIL="someone@example.com"
+export MY_NAME="$(whoami) <$MY_MAIL>"
+
+# Project Information ######################################## (edit this line)
+#   - project account for computing time
+export proj="<yourProject>"
+
+# Text Editor for Tools ###################################### (edit this line)
+#   - examples: "nano", "vim", "emacs -nw", "vi" or without terminal: "gedit"
+#export EDITOR="nano"
+
+# General modules #############################################################
+#
+module load dgx
+module swap PrgEnv-intel PrgEnv-gnu
+module load cuda openmpi
+module load cmake/3.18.2
+module load boost/1.70.0
+
+# Other Software ##############################################################
+#
+# module load cray-hdf5-parallel/1.10.2.0  # todo: probably wrong compiler, linker issues
+module load png/1.6.34
+
+export ADIOS2_ROOT=${HOME}/sw/adios-2.7.1-a100
+export openPMD_ROOT=${HOME}/sw/openPMD-api-0.13.4-a100
+export PNGwriter_ROOT=${HOME}/sw/pngwriter-0.7.0-21-g9dc58ed
+
+# Environment #################################################################
+#
+export CC="$(which gcc)"
+export CXX="$(which g++)"
+export CUDACXX=$(which nvcc)
+export CUDAHOSTCXX=$(which g++)
+
+export PICSRC=$HOME/src/picongpu
+export PIC_EXAMPLES=$PICSRC/share/picongpu/examples
+export PIC_BACKEND="cuda:80"
+
+export PATH=$PATH:$PICSRC
+export PATH=$PATH:$PICSRC/bin
+export PATH=$PATH:$PICSRC/src/tools/bin
+
+export LD_LIBRARY_PATH=$ADIOS2_ROOT/lib64:$openPMD_ROOT/lib64:$PNGwriter_ROOT/lib64:$LD_LIBRARY_PATH
+export PYTHONPATH=$PICSRC/lib/python:$PYTHONPATH
+
+# "tbg" default options #######################################################
+#   - SLURM (sbatch)
+#   - "defq" queue
+export TBG_SUBMIT="sbatch"
+export TBG_TPLFILE="etc/picongpu/cori-nersc/a100.tpl"
+
+# allocate an interactive shell for one hour
+#   getNode 2  # allocates two interactive A100 GPUs (default: 1)
+function getNode() {
+    if [ -z "$1" ] ; then
+        numGPUs=1
+    else
+        numGPUs=$1
+    fi
+    srun --time=1:00:00 --nodes=1 --ntasks-per-node $numGPUs --gpus $numGPUs --cpus-per-task=8 -A $proj -C dgx -q shared --pty bash
```

Comment:
Hackathon: add reservation
```suggestion
    srun --time=1:00:00 --nodes=1 --ntasks-per-node $numGPUs --gpus $numGPUs --cpus-per-task=8 -A $proj -C dgx -q shared --reservation="gpu_hackathon_dgx" --pty bash
```
(not to be committed to docs, because specific to our event)